### DISCUSSION TITLE
Is this a bug (Settings/Timer Processing)?

### DISCUSSION THREAD

**Discussion by aumusic** [dialogic-godot/dialogic#1868] [General]
I've been meaning to write about this one for a while, but I keep forgetting about it, and since I'm not certain whether it is a bug or something I'm overlooking (and I'm not certain it carries over into the latest Alpha, though I haven't seen any notes on it), I thought I'd post it here rather than the 'issues' section...

I've noticed that, in Alpha 9, whenever I try to activate the "Timer processing" check-box within the Settings menu, Godot generates the following error, and when the Settings menu is refreshed, the check-box is deselected:
  core/object/object.cpp:1082 - Error calling from signal 'pressed' to callable: 'VBoxContainer(settings_general.gd)::_on_physics_timer_button_toggled': Method expected 0 arguments, but called with 0.

Any thoughts?

**Comment by CakeVR** [comment]
Thank you for asking this question, you found a bug!

Feel free to try this fix: https://github.com/coppolaemilio/dialogic/pull/1872
Editor needs to reload, e.g. by restarting Godot.

**Reply by aumusic** [reply]
@CakeVR , thank you for this, but I found it still generated the same error for me.

EDIT: Never mind; it does work. For some reason, when I initially viewed the changes, I only saw the modified line 111 and not the modified line 23. Sorry about the mix-up.

**Reply by CakeVR** [reply]
That's nice, glad I was able to help you! If you want, you can mark this as solved! :)
